### PR TITLE
IDE-4965 upload files to linode instead of to bintray

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

File: build/com.liferay.ide-repository/deployToS3.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+if [ $# != 4 ]
+then
+  echo "Usage: $0 repo_path bt_repo bt_package bt_version"
+  exit 1
+fi
+
+REPO_PATH=$1
+BT_REPO=$2
+BT_PACKAGE=$3
```
Comment (gamerson) [inline build/com.liferay.ide-repository/deployToS3.sh:15]:
```suggestion
PACKAGE=$3
```

File: build/com.liferay.ide-repository/deployToS3.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+BT_VERSION=$4
```
Comment (gamerson) [inline build/com.liferay.ide-repository/deployToS3.sh:16]:
```suggestion
VERSION=$4
```

File: build/osgi-bundle-repository/deployToS3.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+echo "please make sure your s3cmd has been configured"
+
+if [ $# != 4 ]
+then
+  echo "Usage: $0 repo_path bt_repo bt_package bt_version"
+  exit 1
+fi
+
+REPO_PATH=$1
+BT_REPO=$2
```
Comment (gamerson) [inline build/osgi-bundle-repository/deployToS3.sh:16]:
```suggestion
REPO=$2
```

File: build/osgi-bundle-repository/deployToS3.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+BT_PACKAGE=$3
```
Comment (gamerson) [inline build/osgi-bundle-repository/deployToS3.sh:17]:
```suggestion
PACKAGE=$3
```

**Comment by yanan** [comment]
I will update all the variable names.

**Pull request closed without merging**